libs/Theseus/src/Theseus.jl

```diff
@@ -1,364 +1,375 @@
 module Theseus
 
 using UnPack
-import Ariadne
+using LinearAlgebra
+import Ariadne: JacobianOperator
+using Krylov
+
+abstract type RKTableau end
 
 # Wrapper type for solutions from Theseus.jl's own time integrators, partially mimicking
 # SciMLBase.ODESolution
 struct TimeIntegratorSolution{tType, uType, P}
-    t::tType
-    u::uType
-    prob::P
+	t::tType
+	u::uType
+	prob::P
 end
```

Comment:
It looks like there are a lot of whitespace changes replacing spaces by tabs. Why?